### ISSUE TITLE
Do user supplied uploads need sanitising?

### ISSUE THREAD

**Issue by DazDotOne** [thephpleague/csv#411]
_(Fill in the relevant information below to help triage your issue.)_

|    Q        |   A
|------------ | ------
| Version     | ^9.6

## Question
_(Please explain in plain english your question)_

I was just wondering whether the package performs any sanitisation behind the scenes?

I'm allowing users to upload a csv (i.e. potentially unsafe input).

Do I need to sanitise the file before I read it with the package or is that performed by the reader?

If I need to perform it, are there any best practices that you are aware of?

Thanks for your time.

## Checks before submitting

* [x] Be sure that there isn't already an issue about this. See: [Issues list](https://github.com/thephpleague/csv/issues)
* [x] Be sure that there isn't already a pull request about this. See: [Pull requests](https://github.com/thephpleague/csv/pulls)
* [x] I have read, search and not found the information on the [documentation website](https://csv.thephpleague.com).
* [x] I have read, search and not found the information on PHP related forums and/or websites.
* [x] This issue is about 1 question around the package with no business or domain specific logic related to a specific situation.
* [x] The question has a descriptive title. For example:  "Can I use the library with compressed CSV documents ?".

**Comment by nyamsprod** [comment]
@DazDotOne thanks for using the library and yes the package does not sanity or validation check these should be performed by you prior to using the document with the library.
For best practices I'm afraid it all depends on how you acquire and interact with the document and which type of document you get. I can't answer that question.

**Comment by DazDotOne** [comment]
Thanks for the help (and package). I figured I ask before starting to work on a solution, 

I'll get started then.
